public/app/features/canvas/runtime/element.tsx

```diff
@@ -358,6 +358,10 @@ export class ElementState implements LayerElement {
       }
     }
 
+    if (border && border.radius !== undefined) {
+      css.borderRadius = `${border.radius}px`;
+    }
```

Comment:
For svg based elements (i.e. ellipse, triangle, parellogram, and cloud?) we need to apply border radius to actual svg (similar to background image etc)

TODO create a follow-up GH issue for this

![Screenshot 2024-03-22 at 11 33 11 AM](https://github.com/grafana/grafana/assets/22381771/6bbf66cb-319b-41a0-b7e7-5045d692fdd4)